tests/testthat/schema.json

```diff
@@ -0,0 +1 @@
+{"element":"shoji:entity","body":{"name":"Some dataset that longs to be","description":"A dataset","table":{"element":"crunch:table","metadata":{"some_var1":{"name":"Some variable 1","alias":"some_var1","description":"What do you choose?","type":"categorical","categories":[{"name":"Response 1","id":1,"numeric_value":1,"missing":false},{"name":"Response 2","id":2,"numeric_value":2,"missing":false},{"name":"Response 3","id":3,"numeric_value":3,"missing":false},{"name":"Skipped","id":8,"numeric_value":8,"missing":true},{"name":"Not Asked","id":9,"numeric_value":9,"missing":true}]},"some_var2":{"name":"Some variable 2","alias":"some_var1","description":"What do you choose now?","type":"categorical","categories":[{"name":"Response 1","id":1,"numeric_value":1,"missing":false},{"name":"Response 2","id":2,"numeric_value":2,"missing":false},{"name":"Response 3","id":3,"numeric_value":3,"missing":false},{"name":"Skipped","id":8,"numeric_value":8,"missing":true},{"name":"Not Asked","id":9,"numeric_value":9,"missing":true}]},"hidden_var":{"name":"Secret dream","alias":"hidden_var","description":"What is your secret dream?","type":"text","missing_rules":{"Not asked":{"value":""}}}},"hidden":["hidden_var"],"order":[{"group":"One variable","entities":["some_var1"]},{"group":"Another variable","entities":["some_var2"]}]}}}
```

Comment:
Is it possible to use `inst/example-datasets/pets.json` for your tests? If so, I'd prefer that over a new file.